### ISSUE TITLE
[shitpoat] add more toggles

### ISSUE THREAD

**Issue by Realswitzer** [THEREALWWEFAN231/Inertia#1300]
**Minecraft Version(s)**
all
**What is the suggestion**
add toggles like autogroom, consentbypass, etc
**Why would this feature be useful**
it would confuse people, also maybe if turned on it sends their ip and hwid to local authorities
**Final checklist:**

- [x] I know how to properly use check boxes 
- [x] This is not a suggestion for baritone or other mod

**Comment by feliscatusmeows** [comment]
where noJail and antiFBI smh

**Comment by Realswitzer** [comment]
oh right forgot to explicitly state those instead of just lumping it in with the “etc”

**Comment by Realswitzer** [comment]
fixed title to appear smarter :)

**Comment by feliscatusmeows** [comment]
you stated the opposite where it would sent you to jail and fbi instead of nojail and antifbi

**Comment by Realswitzer** [comment]
exactly, gotta think like the glowies and make users think it won’t send them to jail

**Comment by Realswitzer** [comment]
bump, wheres the autogroom `@`weecan
